Components/card.html

```diff
@@ -0,0 +1,304 @@
+    />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="/docs.css" />
+    <link rel="stylesheet" href="/Components-CSS/button.css" />
+    <link rel="stylesheet" href="/Components-CSS/badge.css" />
+    <link rel="stylesheet" href="/Components-CSS/card.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="header">
+        <a class="logo" href="/index.html">
+          <img width="80" src="/images/music.svg" alt="logo" />
+          <span><strong> RHYTHM UI</strong></span>
+        </a>
+        <div class="search-container">
+          <div class="search-box search">
+            <i class="fas fa-search"></i>
+
+            <span>
+              <input type="text" name="" placeholder="Search..." />
+            </span>
+          </div>
+        </div>
+        <a class="home" href="/">
+          <i class="fas fa-home"></i>
+
+          Home</a
+        >
+        <a class="docs active" href="/docs.html">
+          <i class="fas fa-file-alt"></i>
+
+          Docs</a
+        >
+        <a
+          class="github"
+          target="blank"
+
+          href="https://github.com/chandana105/rhythm-ui-component-library"
+          ><i class="fab fa-github"></i>
+
+          Github</a
+        >
+      </nav>
+      <div class="side-bar">
+        <ul>
+          <li>
+            <a href="/docs.html">Getting Started</a>
+          </li>
+          <li>
+            <a href="/Components/avatar.html">Avatar</a>
+          </li>
+          <li>
+            <a href="/Components/alert.html">Alert</a>
+          </li>
+          <li>
+            <a href="/Components/badge.html">Badge</a>
+          </li>
+          <li>
+            <a href="/Components/button.html">Button</a>
+          </li>
+          <li>
+            <a href="/Components/card.html" class="active">Card</a>
+          </li>
+          <li>
+            <a href="/Components/image.html">Image</a>
+          </li>
+          <li>
+            <a href="/Components/input.html">Input</a>
+          </li>
+          <li>
+            <a href="/Components/lists.html"> Lists</a>
+          </li>
+          <li>
+            <a href="/Components/navigation.html">Navigation</a>
+          </li>
+          <li>
+            <a href="/Components/modal.html">Modal</a>
+          </li>
+          <li>
+            <a href="/Components/rating.html">Rating</a>
+          </li>
+          <li>
+            <a href="/Components/grid.html"> Simplified Grid</a>
+          </li>
+          <li>
+            <a href="/Components/typography.html">Text Utilities</a>
+          </li>
+          <li>
+            <a href="/Components/snackbar.html">Snackbar</a>
+          </li>
+        </ul>
+      </div>
+      <div class="content">
+        <main>
+          <h1>Cards</h1>
+          <br />
+
+          <div class="card-container">
+            <h3>Card with badge</h3>
+            <br />
```

Comment:
This may not be a good practice to use the ```br``` in such a way.

Ref this: https://stackoverflow.com/questions/462619/should-br-and-hr-be-avoided-at-all-costs-in-web-design/462703